GenerateOriginalFiles.py

```diff
@@ -230,5 +232,8 @@ def CopyIncludeAndPortableDirs():
     shutil.copytree('source/portable', 'portable')
 
 if __name__ == "__main__":
+    # Move the working directory to the root of repository.
```

Comment:
```suggestion
    # Change the working directory to the root of repository.
```
Nit: since "move" implies `mv` rather than `cd`

Reply:
Done.